framework/src/wifi_manager/wifi_manager_profile.h

```diff
@@ -19,6 +19,8 @@
 #ifndef _WIFI_PROFILE_H__
 #define _WIFI_PROFILE_H__
 
+#include <tinyara/wifi/wifi_common.h>
+
```

Comment:
it's not recommended to include header files in header file. because it causes circular dependencies. so please include wifi_common.h to c file.

Reply:
done